power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/short_circuit_solver.hpp

```diff
@@ -260,20 +362,30 @@ class ShortCircuitSolver {
         }
     }
 
-    void assert_all_fault_type_phase_equal_(const std::vector<FaultCalcParam>& vec) {
-        assert(!vec.empty());  // Assert that the vector is not empty
+    bool all_fault_type_phase_equal_(const std::vector<FaultCalcParam>& vec) {
```

Comment:
why do you need the trailing underscore `_`? is there another name conflicting with this one?

Reply:
Nope, was a mistake. Fixed